changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## `v0.16.0`
+
+### Features:
+
+- Added ability to pass a genesis file and peer address to `starport network chain join`
+- Added ability to show genesis or peer list with a flag in `starport network chain show`
+- `protoc` is now bundled with Starport CLI
+- Starport is now published automatically on the Docker Hub
+- Added Keplr extension support to the web app
+- Added IBC relayer support to the web app
+- Added to scaffold a type without CRUD operations with `--no-message` in the `type` command
```

Comment:
```suggestion
- New types without CRUD operations can be added with the `--no-message` flag in the `type` command
```

Reply:
what is the benefit of types w/o CRUD?

Reply:
When you want a type, behavior of which doesn't match standard CRUD operations.

Reply:
E.g. it's not an "object" you can create/delete, but something else that you can send/process/[any other verb].